sdk/util/rotation.h

```diff
@@ -126,6 +126,14 @@ class Rotation {
   // Multiply a Rotation and a Vector to get a Vector.
   VectorType operator*(const VectorType& v) const;
 
+  // Functions that return the Yaw, Pitch and Roll angle from the current 
```

Comment:
Consider creating different comment headers for each method. Consider also applying the following wordsmithing:

Computes the Euler [name] angle (rotation around the [letter]-axis) given the current value of @p quat_. @see https://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles for further details.

@return <name> angle in radians.

Reply:
Consider marking the methods as const.

Reply:
Nit: `@p`  is used for parameters, in this case there is none.

I think it'll be super simple if you say that each angle is the rotation along an axis in particular. That would allow readers to easily understand which is the angle we are referring to.

Reply:
Also, explain the range in radians for each angle.

I recommend to use π where appropriate.